src/diff/props.js

```diff
@@ -49,7 +49,7 @@ function setStyle(style, key, value) {
  * @param {string} name The name of the property to set
  * @param {*} value The value to set the property to
  * @param {*} oldValue The old value the property had
- * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
+ * @param {number} isSvg 0 if not an SVG element, else it is an SVG element
```

Comment:
This parameter is only needed to convert `xlink;href`/`xlinkHref` to `href` and `className` to `class`. I wonder if there is a smaller way to determine this behavior without needing this param. Perhaps drop support for `xlink:href` and `xlinkHref` by providing a codemod and figure something else out for `className` -> `class`...

Something to think about for a future PR